script/functions.sh

```diff
@@ -6,7 +6,7 @@ source $SCRIPT_DIR/travis_functions.sh
 source $SCRIPT_DIR/predicate_functions.sh
 
 # If JRUBY_OPTS isn't set, use these.
-# see http://docs.travis-ci.com/user/ci-environment/
+# see https://docs.travis-ci.com/user/reference/overview/
```

Comment:
This is where the old link redirects right now. It doesn't mention `JRUBY_OPTS` or env variables at all, so I'm wondering if that's the correct link?

Reply:
It'll do